beacon/endpoints/rest/schemas/default.py

```diff
@@ -109,22 +110,38 @@ def beacon_biosample_v20(row):
             'tumorProgression': row['tumor_progression_ontology'],
             'tumorGrade': row['tumor_grade_ontology'],
         },
-        'info': None
+        'handovers': [jsonb(h) for h in row['handovers']],
+        'info': {
+            'alternativeIds': row['alternative_ids'],
+            'studyId': row['study_id'],
+            'bioprojectId': row['bioproject_id'],
+            'files': [jsonb(v) for v in row['files'] or []],
+        }
     }
 
 
 def beacon_individual_v20(row):
     schema_name = 'beacon-individual-v2.0.0-draft.2'
     return {
```

Comment:
Looks like we're getting closer and closer to you formatting _everything_ in the DB, and we only let python print the result. That's super nice and efficient, but a little hard to update...

Reply:
Yes, I've also thought about this. The Python code is cleaner and more efficient but the Postgres functions are getting bigger and more complicated.